azurerm/resource_arm_managed_disk.go

```diff
@@ -102,6 +102,18 @@ func resourceArmManagedDisk() *schema.Resource {
 			"encryption_settings": encryptionSettingsSchema(),
 
 			"tags": tagsSchema(),
+
+			"disk_iops_read_write": {
+				Type:     schema.TypeInt,
+				Optional: true,
```

Comment:
I personally would remove the `Optional` attribute from this parameter since it is a `read-only` property of the resource... what do you think?

Reply:
Thanks for your review. I think this property is not read-only. Because it can be specified to update the number of iops while using ultra ssd disk. You may want to refer the document and search by keyword "properties.diskIOPSReadWrite" which describes the usage of this property.